frontend/packages/noobaa-storage-plugin/src/components/health-card/health-card.tsx

```diff
@@ -53,13 +52,16 @@ const getObjectStorageHealthState = (
   const unhealthyBuckets = getGaugeValue(unhealthyBucketsResponse);
   const pools = getGaugeValue(poolsResponse);
   const unhealthyPools = getGaugeValue(unhealthyPoolResponse);
+
   const result: ObjectStorageHealth = {
     message: 'Object Storage is healthy',
     state: HealthState.OK,
   };
 
-  let value;
-  if (_.isEmpty(noobaaSystemData)) {
+  const noobaaSystemData = _.get(noobaaSystem, 'data[0]', null) as K8sResourceKind;
+  const noobaaPhase = _.get(noobaaSystemData, 'status.phase');
```

Comment:
```suggestion
  const noobaaPhase = _.get(noobaaSystemData, 'status.phase', '');
```

Reply:
Would still give a falsey value. `''` is falsey which would make it equivalent to using it as it is since `undefined` is also a falsey value.